sphinx_rtd_theme/layout.html

```diff
@@ -131,7 +131,7 @@
             {#- Not strictly valid HTML, but it's the only way to display/scale
                 it properly, without weird scripting or heaps of work
             #}
-            <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="{{ _('Logo') }}"/>
+            <img src="{{ logo_url }}" class="logo" alt="{{ _('Logo') }}"/>
```

Comment:
This will fail users of older sphinx versions use an updated version of the theme. You will have to do a version comparison and keep both methods available. I am going to put a hold on this PR until we can figure out how to do proper tuple comparisons; see https://github.com/readthedocs/sphinx_rtd_theme/pull/1123